Kudu.Tests/Core/Function/KafkaTriggerKedaAuthProviderTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Kudu.Core.Functions;
```

Comment:
I can find two similar test.  Why?

Kudu.Tests/Core/Function/KafkaTriggerKedaAuthProviderTests.cs
Kudu.Tests/Core/Function/KafkaTriggerKedaAuthProviderTest.cs

Reply:
Missed the commit to delete Kudu.Tests/Core/Function/KafkaTriggerKedaAuthProviderTest.cs . Fixed now.